core/src/main/java/nl/nn/adapterframework/core/PipeLine.java

```diff
@@ -652,7 +652,6 @@ public void setOutputWrapper(IWrapperPipe outputWrapper) {
 		this.outputWrapper = outputWrapper;
 	}
 
-	@IbisDoc({ "PipeLine exits. If no exits are specified, a default one is created with path=\""+DEFAULT_SUCCESS_EXIT_NAME+"\" and state=\"SUCCESS\""})
```

Comment:
This ibisdoc should not be removed, as it is displayed in https://frankdoc.frankframework.org/#!/Other/PipeLine.

Reply:
I have undone deleting it.